pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "bids2openminds"
+version = "0.0.0"
+dependencies = [
+  "bids",
+  "openminds",
```

Comment:
I think we should add a openminds version as only the latest one can use instances 
`"openminds >= 0.2.0 ",`

Reply:
More important: v2 and v1 have errors. so we should use the bids2openminds only for openminds v3 and higher

Reply:
@lzehl the target openMINDS version could be added as a command-line option: @Peyman-N could you create an issue for that?

For the dependency on openMINDS-Python, I'm making a new release 0.2.1 which includes the improved IRI validation.

Reply:
ok, @Peyman-N that's done (>= 0.2.1)